app/javascript/groceries/components/StoreHeader.vue

```diff
@@ -1,5 +1,5 @@
 <template lang="pug">
-h2.store-name.my-4
+h1.h2.store-name.my-4
```

Comment:
This is why I made the rules above apply not only to e.g. `h2` by default but also to `.h2` elements.

Reply:
This should be an `h1` tag because it's the main title of the page. This was surfaced to me by writing the spec first for the change (`expect(page).to have_css('h1', text: existing_store.name)`).